qmetaobject/src/qtdeclarative.rs

```diff
@@ -926,13 +922,29 @@ cpp_class!(
     pub unsafe struct QJSValue as "QJSValue"
 );
 
+/// Wrapper for [`QJSValue::SpecialValue`][qt]
+///
+/// [qt]: https://doc.qt.io/qt-5/qjsvalue.html#SpecialValue-enum
+#[repr(u32)]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum QJSValueSpecialValue {
```

Comment:
I don't know if it makes sense to expose this in rust.

I would prefer having functiopn like 

`QJSValue::null()`  to create a null value, and `QJSValue::default()` creates an undefined value  (we could also add `undefined()` 

What do you think?

Reply:
But public and documented in Qt doesn't mean there is  reason to expose them in this crate.

Reply:
:thinking: konsistency? After all, what are the downsides?

I have to admit, I'm not a big fan of a lengthy and repetitive name `QJSValueSpecialValue`, but that's due to a technical restriction in Rust. Maybe `QJSSpecialValue` would be marginally better, since it's not 1:1 mapping anyway?

Reply:
I agree, that `null` and `undefined` could be useful.

However, I also think having consistency with Qt Framework is important. If someone decides to use the crate, they would expect a 1:1 mapping of entities, that come from Qt. They may also find the enum useful (for some reason).